Tests/FoundationInternationalizationTests/Formatting/DateFormatStyleTests.swift

```diff
@@ -29,12 +29,6 @@ import TestSupport
 import _RopeModule
 #endif
 
-extension AttributedString {
-    fileprivate var string: String {
-        String(self._guts.string)
```

Comment:
Since we can no longer `@testable import FoundationEssentials`, we can no longer access `_guts`. Let's just update the test to not require this.